drivers/classic/src/initialize_atmos.c

```diff
@@ -101,39 +100,44 @@ initialize_atmos(atmos_data_struct    *atmos,
     double                     delta_t_plus;
     int                        have_dewpt;
     int                        have_shortwave;
-    int                        hour_offset_int;
-    int                        tmp_starthour, tmp_endhour;
-    short                      local_startyear;
-    short                      local_startmonth;
-    short                      local_startday;
-    short                      local_starthour;
-    unsigned short             day_in_year, year, month, days_in_month;
+    int                        sec_offset_int;
```

Comment:
By the way, I'm also reviewing the code (Joe asked me to).  But I'm not
seeing my comments show up in this email thread, strangely.  Are you seeing
my comments?

Ted

On Mon, Jan 12, 2015 at 9:09 PM, Ted Bohn <ted.j.bohn@gmail.com> wrote:

> I think Joe was simply maintaining the old variable name (was originally
> hour_offset_int).  I had created hour_offset_int to distinguish it from
> hour_offset.  hour_offset_int was the hour_offset, rounded to the nearest
> hour, for the purpose of computing the fractional hour between local solar
> time and "official" time in that time zone.
>
> On Mon, Jan 12, 2015 at 9:04 PM, Bart Nijssen <notifications@github.com>
> wrote:
>
>> In drivers/classic/src/initialize_atmos.c
>> <https://github.com/UW-Hydro/VIC/pull/188#discussion-diff-22842408>:
>>
>> > @@ -101,39 +100,44 @@ initialize_atmos(atmos_data_struct    *atmos,
>> >      double                     delta_t_plus;
>> >      int                        have_dewpt;
>> >      int                        have_shortwave;
>> > -    int                        hour_offset_int;
>> > -    int                        tmp_starthour, tmp_endhour;
>> > -    short                      local_startyear;
>> > -    short                      local_startmonth;
>> > -    short                      local_startday;
>> > -    short                      local_starthour;
>> > -    unsigned short             day_in_year, year, month, days_in_month;
>> > +    int                        sec_offset_int;
>>
>> Remove the _int from the variable name. This is not a coding style that
>> has been followed in VIC, so having it for one or two variables only does
>> not help. Note there are variable naming conventions that people used to
>> follow ("Hungarian"), in which every integer started with an i, every
>> pointer with a p, but it is actually difficult to be consistent in this
>> (point to an array of structures, etc).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/UW-Hydro/VIC/pull/188/files#r22842408>.
>>
>
>